src/build.py

```diff
@@ -115,6 +114,7 @@
 GITHUB_MIRROR_BASE = GIT_MIRROR_BASE + 'external/github.com/'
 WASM_GIT_BASE = GITHUB_MIRROR_BASE + 'WebAssembly/'
 EMSCRIPTEN_GIT_BASE = GITHUB_MIRROR_BASE + 'kripken/'
+LLVM_GIT_BASE = 'https://github.com/llvm/'
```

Comment:
Should we wait for https://llvm.googlesource.com/ to mirror this repo?

Reply:
Turns out it's already available on https://github.googlesource.com/llvm/llvm-project/, I'll update.

Reply:
Huh, it's been around for a while... how have I never heard of this mirror?

Reply:
Seems like it does: https://github.googlesource.com/WebAssembly/wabt

Looks like most/all of them: https://github.googlesource.com/WebAssembly/

Reply:
Oh, indeed it does! In other news the repo search on that homepage is apparently case sensitive ಠ_ಠ